packages/tabs/src/useTabs.ts

```diff
@@ -9,16 +9,46 @@ import { useState } from 'react';
 import { useSelection } from '@zendeskgarden/container-selection';
 import { generateId } from '@zendeskgarden/container-utilities';
 
-function requiredArguments(arg, argStr, methodName) {
+interface IUseTabsPropGetters {
+  getTabProps: <T>(options?: T & IGetTabPropsOptions) => T & IGetTabPropsOptions;
+  getTabPanelProps: <T>(options?: T & IGetTabPanelProps) => IGetTabPanelProps;
+  getTabListProps: <T>(options: T) => T & React.HTMLProps<any>;
+}
+
+interface IGetTabPropsOptions extends React.HTMLProps<any> {
```

Comment:
This should extend the `IGetItemPropsOptions<Item>` interface

Reply:
Updated, thanks!